_episodes/07-cond.md

```diff
@@ -63,7 +63,7 @@ Python simply does nothing if the test is false:
 num = 53
 print('before conditional...')
 if num > 100:
-    print(num,' is greater than 100')
+    print(num,'is greater than 100')
```

Comment:
Good catch, Nancy! Would you mind also adding in a space between the comma and quote, for consistency with the spacing used in the lesson?

Reply:
Done! This is my first time contributing this way. Does the additional commit and this comment suffice or should I do something more? Thanks!

Reply:
What you did is what I had hoped for. Thank you!